### PR TITLE
Add test/case for null outputs

### PR DESCRIPTION
**Pull request by gnarf**

### CODE REVIEW

File: src/__snapshots__/shared-render.test.js.snap [hunk 1]
```diff
@@ -23,6 +23,15 @@ preact-render-spy (3 nodes)
...
 exports[`deep: output matches snapshot 1`] = `<div>second first</div>`;
 
+exports[`deep: weird render cases toString matches snapshot 1`] = `
+"preact-render-spy (1 nodes)
+-------
+{null}
+"
+`;
+
+exports[`deep: weird render cases toString matches snapshot 2`] = `preact-render-spy (0 nodes)`;
```
Comment (mzgoddard) [inline src/__snapshots__/shared-render.test.js.snap]: :heart:
